lib/data.js

```diff
@@ -1,11 +1,19 @@
+import { uid } from "uid";
+
 export const initialProducts = [
-  {
-    name: "Bread",
-  },
   {
     name: "Milk",
+    _id: uid(),
+    note: "Lorem ipsum dolore sit amet ",
   },
```

Comment:
Using the `uid()` function in a static can lead to new IDs being generated every time the array is initialized and this can cause several issues e.g. IDs used as keys can lead to unnecessary re-renders which impacting performance.

➡️ _So, it's advisable to use static, manually assigned IDs for static datasets, use `uid()` only when dynamically generating data._


```suggestion
  {
    name: "Milk",
    _id: "1",
    note: "Lorem ipsum dolore sit amet ",
  },
```

Reply:
Makes sense. We removed the uid from the data.js file